scripts/e2e-ci.sh

```diff
@@ -389,6 +406,9 @@ test_exec() {
 
     # train tests
     if [ "$LARGE" -eq 1 ]; then
+      # Validate a single epoch per phase with resumption
```

Comment:
Nit, we are running two epochs in each phase of phased training: https://github.com/instructlab/instructlab/blob/main/scripts/test-data/train-profile-l40s-x4.yaml#L23

Reply:
I think I am overriding it with --phased-phase1-num-epochs in phased_training_resume.py ?

Reply:
Oh you're right, I missed that.